python/zygote.py

```diff
@@ -72,9 +72,10 @@
 # debug the problem.
 def try_dumps(obj, sort_keys=False, allow_nan=False):
     try:
+        prairielearn.all_integers_within_limits(obj)
         return json.dumps(obj, sort_keys=sort_keys, allow_nan=allow_nan)
-    except Exception:
-        print(f"Error converting this object to json:\n{obj}\n")
+    except Exception as err:
+        print(f"Error: {err}\n When converting this object to json:\n{obj}\n")
```

Comment:
Is printing the error actually necessary? I believe this will happen automatically when the exception bubbles up and the process dies, and I don't want to print it twice.

Reply:
I'll play around by intentionally triggering this. I think you're right though, not sure why this just catches + re-throws. Though frankly this file is a little weird overall.

Reply:
The catch + rethrow is so we can log the entire JSON object for instructor debugging.

Reply:
Ahhh I see what you mean now, will remove.